app/app.go

```diff
@@ -461,12 +460,8 @@ func NewDesmosApp(
 		wasm.NewAppModule(app.appCodec, &app.WasmKeeper, app.StakingKeeper, app.AccountKeeper, app.BankKeeper, app.MsgServiceRouter(), app.GetSubspace(wasmtypes.ModuleName)),
 	}
 
-	app.RegisterModules(legacyModules...)
-	cfg := app.Configurator()
-	for _, m := range legacyModules {
-		if s, ok := m.(module.HasServices); ok {
-			s.RegisterServices(cfg)
-		}
+	if err := app.RegisterModules(legacyModules...); err != nil {
+		panic(err)
 	}
```

Comment:
`RegisterServices` now is covered inside `RegisterModules`, the manually services setting can be removed safely, or it would panic since the double setting.

See:
https://github.com/cosmos/cosmos-sdk/pull/17284